HOW_TO_TEST.md

```diff
@@ -15,17 +15,26 @@ If the environment has already been setup, you can skip to [Send data](#send-dat
 
 - see https://serverless.com/framework/docs/providers/aws/guide/credentials/
 
-You mainly need to get your AWS keys and set them up by using:
+Firstly, you need to have a AWS account, and a pair of keys associated to it
+- we use a shared AWS account, with several users
```

Comment:
Here we can add something like "Please contact the account manager this shared AWS account to create an account for you". Since Laurent (currently) need to be involved for the user creation.